docs/pages/Configuration.md

```diff
@@ -308,13 +308,14 @@ All changes are documented below.
 
 ### Compatibility Matrix
 
-Unless otherwise noted in the table below, the UI5 Tooling modules are backward compatible in the means that for example UI5 CLI v2.0 will still be able to handle a project that is using Specification Version `1.0`.
+Unless otherwise noted in the table below, the UI5 Tooling modules are backward compatible in the means that for example UI5 CLI v3.0.0 might still be able to handle a project that is using Specification Version `1.0`.
```

Comment:
I'd suggest to modify this sentence as follows:
"Unless otherwise noted in the table below, the UI5 Tooling modules are backward compatible."

Reply:
Thanks, we'll change it to match your proposal in a separate Pull Request 👍